cetmix_tower_server_queue/models/cx_tower_file.py

```diff
@@ -0,0 +1,157 @@
+# Copyright (C) 2025 Cetmix OÜ
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+import logging
+
+from odoo import _, fields, models
+from odoo.exceptions import UserError
+
+_logger = logging.getLogger(__name__)
+
+
+class CxTowerFile(models.Model):
+    _inherit = "cx.tower.file"
+
+    is_being_processed = fields.Boolean(
+        copy=False,
+        help="True if the file is currently being processed",
```

Comment:
Opt for using less technical statements in UI because it's meant for non technical people.
`help="File is currently being processed"`